packages/viz/loadtests.js

```diff
@@ -6,6 +6,13 @@ enzyme.configure({
   disableLifecycleMethods: true,
 });
 
+window.config = {
+  TEST: true,
+  DEV: true,
+};
+// Enable bows logging display:
+// window.localStorage.setItem('debug', 'true');
```

Comment:
Sometimes when launching the unit tests, having the bows logs if useful.
Dev can un-comment it when they need to debug the unit test.

Just to know it can be done, I would have let it commented here.
If it's OK for you